spec/lib/secure_headers_spec.rb

```diff
@@ -60,6 +60,24 @@ module SecureHeaders
         expect(hash["X-Frame-Options"]).to be_nil
       end
 
+      it "Overrides the current default config if default config changes during request" do
```

Comment:
This test exercises the bug in https://github.com/twitter/secureheaders/issues/380.